spec.html

```diff
@@ -7535,6 +7534,17 @@ <h1>SetFunctionName ( _F_, _name_ [ , _prefix_ ] )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-setfunctionlength" aoid="SetFunctionLength">
+      <h1>SetFunctionLength ( _F_, _length_ )</h1>
+      <p>The abstract operation SetFunctionLength requires a Function argument _F_, and a Number argument _length_. This operation adds a `length` property to _F_ by performing the following steps:</p>
+      <emu-alg>
+        1. Assert: _F_ is an extensible object that does not have a `length` own property.
```

Comment:
Should this also assert on IsCallable? Should it call the internal IsExtensible methodminstead of using “is extensible” prose?

Reply:
I copied the proses used elsewhere in the spec word for word. I suggest changing all of it in a separate pull request.

Reply:
Sounds good :-)